### DISCUSSION TITLE
Deployment policy

### DISCUSSION THREAD

**Discussion by wjeziers** [argoproj/argo-cd#6621] [Q&A]
In my organisation we are using argocd and for most cases working fine, but with one except.

This one specific app must be deployed to the kubernetes always when pipeline (on the gitlab) creates a new docker image (with new id) and push them to the repo. 

How can I configure behavior like this ?

**Comment by jannfis** [comment]
Hi @wjeziers, you could take a look whether the [Argo CD Image Updater](https://github.com/argoproj-labs/argocd-image-updater) satisfies your requirements.
